airflow/triggers/base.py

```diff
@@ -124,6 +124,11 @@ def __repr__(self) -> str:
         classpath, kwargs = self.serialize()
         return self.repr(classpath, kwargs)
 
+    def __eq__(self, other: Any) -> bool:
```

Comment:
Where do we need this?

Reply:
That's for a test. Without it, the test fails, I dont remember which one though. I'll remove it to find out. We can see from there if it should be addressed that way

Reply:
Well the test no longer fail so I removed it, I am glad you pointed it out